### DISCUSSION TITLE
Datagrid Pagination Does not Work in a Accordian

### DISCUSSION THREAD

**Discussion by vouxiong** [vmware/clarity#5910] [Q&A]
I was able to get a clr-datagrid working inside a clr-accordian but however if you decide you want pagination on the data grid doesn't display when you toggle a accordion? Is this a bug? 

        <clr-accordion clrAccordionMultiPanel="true">
            <clr-accordion-panel class="audit-only" id="unbalancedNoEdits">
                <clr-accordion-title style="max-width:100%;">
                    <div class="clr-row">
                        <div class="clr-col-4">Bills</div>
                    </div>
                </clr-accordion-title>
                <clr-accordion-content>
                    <clr-datagrid>
                        <clr-dg-column>Bill ID</clr-dg-column>
                        <clr-dg-column>Account Number</clr-dg-column>
                        <clr-dg-column>Site Name</clr-dg-column>
                        <clr-dg-row *ngFor="let bill of billsWithNoOpenTasksAuditOnly?.bills">
                            <clr-dg-cell>
                                <a (click)="onSubAccountBillClicked($event, bill.accountEventId)">{{bill.accountEventId}}</a>
                            </clr-dg-cell>
                            <clr-dg-cell>
                                {{bill.accountNumber}}
                            </clr-dg-cell>
                            <clr-dg-cell>
                                {{bill.clientAssets?.length > 0 ? bill.clientAssets[0]?.clientAssetName : ''}}
                            </clr-dg-cell>
                        </clr-dg-row>
                        <clr-dg-footer>
                            <clr-dg-pagination #pagination [clrDgPageSize]="10">
                                <clr-dg-page-size [clrPageSizeOptions]="[10,20,50,100]">Users per page</clr-dg-page-size>
                                {{pagination.firstItem + 1}} - {{pagination.lastItem + 1}} of {{pagination.totalItems}} users
                            </clr-dg-pagination>
                        </clr-dg-footer>
                    </clr-datagrid>
                </clr-accordion-content>
            </clr-accordion-panel>
        </clr-accordion>

**Comment by gnomeontherun** [comment]
Could you provide us with a functional demo of this so we can review it using Stackblitz? https://stackblitz.com/@clr-team
